app/assets/javascripts/requests/new.js

```diff
@@ -0,0 +1,22 @@
+$(document).on('turbolinks:load', function() {
+  var checkbox_input = $('form#new_request .form-group div #request_time_by_shifts');
+  // Toggle fields initially if checkbox is present
+  if (checkbox_input.length) {
```

Comment:
What does this check? If it has a length? is that the same as checking for presence?

Reply:
In Javascript, `0` is falsey